### DISCUSSION TITLE
Feature request: A11Y Pill Option

### DISCUSSION THREAD

**Discussion by GashyDev** [FortAwesome/Font-Awesome#20938] [Feature Requests]
### Feature description

Hey guys, been a user of FA for many years.

After delving into A11Y a lot more in the past year, I was wondering if there could be another pill option when copying the icon html snippet - something called "A11Y", and inside this would have the mark up in an accessible friendly manner such as `<span aria-hidden="true" class="fa-solid fa-icons"></span>`.

I know accessibility is referenced in the documentation and it is usually down to the developer to implement it, but an option for this feels like a win-win and helps promote the standard for it!

### Additional context

Original
![image](https://user-images.githubusercontent.com/34476863/194049837-558edb53-c035-4644-970e-77d825480f7f.png)

Suggestion
![image](https://user-images.githubusercontent.com/34476863/194050688-e6ace78b-6e35-4637-8251-8b593082c2d3.png)

### Feature request checklist

- [X] The title starts with "Feature request: " and is followed by the requested feature description
- [X] This is a single feature
- [x] I have [searched for existing issues](https://github.com/FortAwesome/Font-Awesome/issues) and to the best of my knowledge this is not a duplicate

**Comment by sensibleworld** [comment]
@GashyDev Thanks so much for the suggestion! This is something we've been thinking a lot about and hope to implement something along these lines at some point (hopefully soon)!

**Comment by tagliala** [comment]
Hi, I know that the teams like `<i>` and it's shorter, but I will take this opportunity to say once again that I personally still think that the default `HTML` documentation should have the accessible version with `<span>` by default, without the need of a specific A11Y pill 😁

**Comment by GashyDev** [comment]
I agree, having it as a default standard would be even better!
I just concepted it as an addition as opposed to converting the base in case there was a reason for the original structure
